### ISSUE TITLE
accessibility IDs are missing from more options

### ISSUE THREAD

**Issue by hegdenitin** [mosip/inji#2424]
**Steps to Re-Produce :**
1. Open Inji wallet
2. Click on + Icon
3. Download Mosip VC from released issuer
4. Click on more options and check accessibility id

 
Actual: accessibility IDs are missing for all the more options button, which is resulting in automation failures
Expected: Accessibility should be restore as it was earlier.

Video:

https://github.com/user-attachments/assets/d1f057de-d30c-4d17-988b-2dd4fc29ed87

test case which is failing: MosipOtpAlternativeFlow.deleteDownloadedVcInOfflineMode
NOTE: all the test cases related to more options are faling. provided one such example above

Report:

[InjiMobileUi-ios-dev-int-inji-2026-04-27-11-50-T-88_P-53_S-0_F-35_KI-0.html](https://github.com/user-attachments/files/27116713/InjiMobileUi-ios-dev-int-inji-2026-04-27-11-50-T-88_P-53_S-0_F-35_KI-0.html)

Screenshot:

<img width="1081" height="703" alt="Image" src="https://github.com/user-attachments/assets/e19004fe-4767-40c8-b976-2b7e215e0033" />

Phone make/model: Realme GT NEO 3T
OS: Android 14
BLE version : 5.2
Applications tested with: 239.apk (0.23.0)

Phone make/model: iPhone 13
OS: iOS 26.4.1
BLE version : 5.2
Applications tested with: 0.23.0 (10)
